doc/iterations/2025.2/add_support_for_border_node_initial_positioning.adoc

```diff
@@ -0,0 +1,32 @@
+= Add Support For Border Node Initial Positioning
+
+== Problem
+
+In a diagram, a `Border Node` is always created on the left of the `Node`(WEST position).
+
+== Key Result
+
+As a specifier, I can define, in the `View` model the side of the `Node` where the `Border Node` will be created.
+
+As an end-user, I can see that the border node is created on the right side of the node accordingly to the view model.
+
+== Solution
+
+Currently, there is only the `NodeDescription` concept used for container, node and border node.
+A `NodeDescription` is considered as the description of a border node if it is contained by the feature `NodeDescription.borderNodeDescriptions`
+
+We need to create a new class `BorderNodeDescription` which is a sub class of `NodeDescription`.
+`BorderNodeDescription` will hold the feature `initialPosition` of type enum `BorderNodePosition`.
```

Comment:
No, it is not the right way to do that.
We should put the information on a xxxLayoutStrategyDescription.
-> to rework

Reply:
The proposed change would have a massive impact and would prevent some use cases that are currently possible. It would be way less impactful to just add the information on the layout strategy description of the parent node. Like in CSS a div can define where its children are using flexbox or grid for example.

Something like this:

```
class LayoutStrategyDescription {
    onRightAtCreationBorderNodes: List<NodeDescription>
    onLeftAtCreationBorderNodes: List<NodeDescription>
    onTopAtCreationBorderNodes: List<NodeDescription>
    onBottomAtCreationBorderNodes: List<NodeDescription>
}
```

We would just need to customize the EMF Edit code to only offer the current border node descriptions of the node description as potential references...